pkg/yurttunnel/server/interceptor.go

```diff
@@ -39,8 +40,23 @@ var (
 	supportedHeaders       = []string{constants.ProxyHostHeaderKey, "User-Agent"}
 	HeaderTransferEncoding = "Transfer-Encoding"
 	HeaderChunked          = "chunked"
+	bufioReaderPool        sync.Pool
 )
 
+func newBufioReader(r io.Reader) *bufio.Reader {
+	if v := bufioReaderPool.Get(); v != nil {
+		br := v.(*bufio.Reader)
+		br.Reset(r)
+		return br
+	}
+	return bufio.NewReader(r)
+}
+
+func putBufioReader(br *bufio.Reader) {
```

Comment:
please add comments for putBufioReader func.

Reply:
done.